admin/google_search_console/class-page-gwt.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * @package WPSEO\admin|google_search_console
+ */
+
+/**
+ * Class WPSEO_Page_GWT
+ */
+class WPSEO_Page_GWT {
+
+	/**
+	 * @var WPSEO_GWT_Service
+	 */
+	private $service;
+
+	/**
+	 * @var WPSEO_GWT_Settings
+	 */
+	private $settings;
+
+	/**
+	 * Constructor for the page class. This will initialize all GSC related stuff
+	 */
+	public function __construct() {
+		// Settings.
+		add_action( 'admin_init', array( $this, 'register_settings' ) );
+
+		// Post to Get on search.
+		add_action( 'admin_init', array( $this, 'list_table_search_post_to_get' ) );
+
+		// Setting the screen option.
+		if ( filter_input( INPUT_GET, 'page' ) === 'wpseo_webmaster_tools' ) {
```

Comment:
maybe the if could be moved to the `page_load` function to keep it scoped to the gwt pages.